tests/api.c

```diff
@@ -1105,6 +1105,170 @@ static int test_cm_load_ca_file(const char* ca_cert_file)
 }
 #endif /* !NO_FILESYSTEM && !NO_CERTS */
 
+static void test_wolfSSL_CertManagerCheckOCSPResponse(void)
+{
+#ifdef HAVE_OCSP
+/* Need one of these for wolfSSL_OCSP_REQUEST_new. */
+#if defined(OPENSSL_ALL) || defined(WOLFSSL_NGINX) || \
+    defined(WOLFSSL_HAPROXY) || defined(WOLFSSL_APACHE_HTTPD) || \
+    defined(HAVE_LIGHTY)
+    WOLFSSL_CERT_MANAGER* cm = NULL;
+    /* Captured with Wireshark using ocsp.test. */
+    byte response[] = {
```

Comment:
Perhaps this should be `const`?

Reply:
This particular one can't be const, unfortunately, because `wolfSSL_CertManagerCheckOCSPResponse` takes the response buffer as non-const.